src/piqtree2/iqtree/_tree.py

```diff
@@ -9,19 +9,74 @@
 
 from piqtree2.exceptions import ParseIqTreeError
 from piqtree2.iqtree._decorator import iqtree_func
-from piqtree2.model import Model
+from piqtree2.model import DnaModel, Model
 
 iq_build_tree = iqtree_func(iq_build_tree, hide_files=True)
 iq_fit_tree = iqtree_func(iq_fit_tree, hide_files=True)
 
+# the order defined in IQ-TREE
+RATE_PARS = ["A/C", "A/G", "A/T", "C/G", "C/T", "G/T"]
+MOTIF_PARS = ["A", "C", "G", "T"]
+
 
 def _rename_iq_tree(tree: cogent3.PhyloNode, names: Sequence[str]) -> None:
     for tip in tree.tips():
         tip.name = names[int(tip.name)]
 
 
-def _process_tree_yaml(tree_yaml: dict, names: Sequence[str]) -> cogent3.PhyloNode:
+def _intrude_edge_pars(tree: cogent3.PhyloNode, **kwargs: dict) -> None:
+    for node in tree.get_edge_vector():
```

Comment:
what's the function purpose, make it clearer that's about mapping edge parameters to cogent3's structure